src/CommunityToolkit/Xamarin.CommunityToolkit/Helpers/LocalizedString.shared.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Xamarin.CommunityToolkit.Helpers
+{
+#if !NETSTANDARD1_0
+	public class LocalizedString : INotifyPropertyChanged
+	{
+		public LocalizedString(Func<string> generator = null)
+            : this(LocalizationResourceManager.Current, generator)
+        {
+        }
+
+		public LocalizedString(LocalizationResourceManager localizationManager, Func<string> generator = null)
+		{
+			Generator = generator;
+			localizationManager.PropertyChanged += (sender, args) => Invalidate();
+		}
+
+		Func<string> generator;
+
+		public Func<string> Generator
+		{
+			get => generator;
+			set
```

Comment:
Do we want that the Generator isn't immutable?

Reply:
Yes, I wanted to give user ability to change generator instead of creating new instance, but maybe this functionality is not needed after all

Reply:
Made generator private